src/server/pps/cmds/cmds.go

```diff
@@ -312,6 +319,69 @@ All jobs created by a pipeline will create commits in the pipeline's repo.
 		}),
 	}
 
+	runPipeline := &cobra.Command{
+		Use:   "run-pipeline pipeline-name [-f job.json]",
+		Short: "Run a pipeline once.",
+		Long:  fmt.Sprintf("Run a pipeline once, optionally overriding some pipeline options by providing a spec.  The spec looks like this:\n%s", exampleRunPipelineSpec),
+		Run: pkgcobra.RunFixedArgs(1, func(args []string) error {
+			apiClient, err := getAPIClient(address)
+			if err != nil {
+				return err
+			}
+			pipelineName := args[0]
+			pipelineInfo, err := apiClient.InspectPipeline(context.Background(), &ppsclient.InspectPipelineRequest{
+				Pipeline: &ppsclient.Pipeline{
+					Name: pipelineName,
+				},
+			})
```

Comment:
I already added `RunPipeline` and `RunPipelineRequest` and it doesn't look too bad.  It will probably scale better as well since we might be adding more options to `RunPipelineRequest` in the future.